src/helpers/git.ts

```diff
@@ -161,18 +161,22 @@ export default class Git {
     }
 
     /**
-     * Rebases the currently checked out branch onto another branch.
+     * Merges the currently checked out branch onto another branch.
      *
      * @param {String} branch The branch to rebase onto.
      * @param {MergeMode} mergeMode
+     * @param {FastForwardMode} fastForwardMode
      *
-     * @throws {Error} if the rebase failed.
+     * @throws {Error} if the merge failed.
      */
-    async merge(branch: string, mergeMode: MergeMode = 'none') {
+    async merge(branch: string, mergeMode: MergeMode = 'none', fastForwardMode: FastForwardMode = "none") {
```

Comment:
Please also add a test to `git.test.ts` for the new `fastForwardMode` setting. :)

Reply:
Hey @saitho thank you for your quick review 😄. Good catch! I completely missed the tests here. As suggested, I have added a few new tests in here for the `fastFowardMode`.